website/content/docs/connect/transparent-proxy.mdx

```diff
@@ -29,8 +29,8 @@ Without transparent proxy, application owners need to:
 
 With transparent proxy:
 
-1. Upstreams are inferred from service intentions, so no explicit configuration
-   is needed.
+1. Upstreams are inferred from service intentions and/or imported services,
```

Comment:
Should we clarify that the intentions we infer upstreams from are the ones from the local cluster and not ones set in remote peers?

Reply:
Yes, that's a good point.